main.tf

```diff
@@ -47,7 +59,7 @@ resource "azurerm_application_insights" "appinsights" {
 
   name                = "${var.product}-appinsights-${var.env}${var.deployment_target}"
   location            = "${var.appinsights_location}"
-  resource_group_name = "${local.resource_group_name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
```

Comment:
please revert this change, we're in the progress of migrating applications to always passing in a resource group to the module, as we're decommissioning this module as part of moving to Kubernetes

REQ

Reply:
https://github.com/hmcts/cnp-module-webapp/pull/156#discussion_r322741012